package.json

```diff
@@ -4,6 +4,7 @@
     "test": "percy exec --testing -- dotnet test"
   },
   "devDependencies": {
-    "@percy/cli": "^1.16.0"
+    "@percy/cli": "1.27.0-alpha.0",
+    "@percy/webdriver-utils": "1.27.0-alpha.0"
```

Comment:
currently there is no test which utilises `webdriver-utils` in tests, its fine for now.

but we'll need to add coverage for POA tests.